esx_service/vmdk_ops.py

```diff
@@ -468,6 +468,10 @@ def findVmByUuid(vm_uuid):
     vm = si.content.searchIndex.FindByUuid(None, vm_uuid, True, False)
     return vm
 
+def vm_uuid2name(vm_uuid):
+    vm = findVmByUuid(vm_uuid)
+    if vm:
+        return vm.config.name
```

Comment:
it does not return anything if `not vm`. It's OK in python (will just return None) but it looks unpleasant. 
Something like this reads easier IMO . at any rate, your call as the result is correct

```python
if not vm or not vm.config:
  return None
return vm.config.name
```